internal/sockstate/netstat_darwin.go

```diff
@@ -0,0 +1,11 @@
+package sockstate
```

Comment:
Should have a `// +build darwin`

Reply:
AFAIK naming the module as mod_OSNAME.go has the same effect (at least Goland and the Go toolchain recognise it). Am I wrong?

Reply:
PS: added anyway, explicit is better.